tapir/shifts/views/management.py

```diff
@@ -280,8 +280,8 @@ def form_valid(self, form):
                     required_capabilities=entry.required_capabilities,
                     warnings=entry.warnings,
                 )
-                shift_template_copy_destination.create_shift(
-                    shift_template_copy_destination.start_date
+                shift_template_copy_destination.group.create_shifts(
+                    get_monday(shift_template_copy_destination.start_date)
```

Comment:
This creates shifts for all the shift templates that belong to the same group as the destination. We want to creates shifts EXCLUSIVELY for the new shift template, as already commented [here](https://github.com/SuperCoopBerlin/tapir/pull/613#discussion_r1998590414) and [here](https://github.com/SuperCoopBerlin/tapir/pull/613#discussion_r1980888108).